### PR TITLE
feat: user can press start in custom chat

### PR DESCRIPTION
**Pull request by huydosgtech**

### CODE REVIEW

**Comment by huydosgtech** [comment]
video demo: 

https://github.com/saigontechnology/rn-firebase-chat/assets/105196859/a9839984-04de-458a-b9de-8c0f32ff0f3e

**Review by honghoangsts — CHANGES_REQUESTED** [review]

File: example/src/navigation/AppNavigator.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React, {useRef} from 'react';
+import {NavigationContainer} from '@react-navigation/native';
+import {StackNavigation} from './StackNavigation';
+import {ChatProvider} from 'rn-firebase-chat';
+
+function AppNavigation(): React.ReactElement {
+  const navigationRef = useRef<any>();
```
[the remark this answers is not in the document]
Comment (huydosgtech) [reply example/src/navigation/AppNavigator.tsx]: yes, I updated. Please help me check it

File: src/chat/ChatProvider.tsx [hunk 2]
```diff
@@ -25,8 +25,7 @@ export const ChatProvider: React.FC<ChatProviderProps> = ({
       });
 
       firestoreServices.listenConversationUpdate((data) => {
-        //TODO: handle update conversation changed
-        console.log(data);
+        dispatch(setConversation(data));
```
Comment (honghoangsts) [inline src/chat/ChatProvider.tsx:28]: `updateConversation`

File: src/chat/components/InputToolbar.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  hasPhoto?: boolean;
+  hasCamera?: boolean;
+}
+
+export const InputToolbar: React.FC<IInputToolbar> = ({
+  messagePlaceholder = 'Type your message ...',
+  ...props
+}) => {
+  return (
+    <View>
```
Comment (honghoangsts) [inline src/chat/components/InputToolbar.tsx:16]: Do we need this `View`

File: src/interfaces/chat.ts [hunk 4]
```diff
@@ -4,6 +4,7 @@ import type { Dispatch } from 'react';
 export interface IUserInfo {
   id: string;
   name: string;
+  memberId: string;
```
Comment (honghoangsts) [inline src/interfaces/chat.ts:7]: why we need add `memberId` to `IUserInfo`

**Pull request closed without merging**